test/e2e/qat/qatplugin_dpdk.go

```diff
@@ -251,6 +255,39 @@ func runCpaSampleCode(ctx context.Context, f *framework.Framework, runTests int,
 	gomega.Expect(err).To(gomega.BeNil(), utils.GetPodLogs(ctx, f, pod.ObjectMeta.Name, pod.Spec.Containers[0].Name))
 }
 
+func runQatEngineTestApp(ctx context.Context, f *framework.Framework, resourceName v1.ResourceName) {
```

Comment:
seems like a lot of duplication with `runCpaSampleCode`

Reply:
Yeah, I was actually thinking about that. Do you think it's good to make a common function that gets a parameter for the `commands`?

Reply:
I made a function, but because of the bad name of the image, idk what name is good for the function...

Reply:
I resolve the conversation.! Please open another comment if needed! :)